docs/contributing.md

```diff
@@ -30,11 +29,11 @@ Contributing to the documentation is welcome, but when making
 changes to documentation, please ensure that all changes are
 made **only** to:
 
-- Updating ``/docs.wrm/*\*.wrm``
-- Adding links: ``/docs.wrm/links/*.txt``
-- Updating API jsdocs: ``/*\* ... */`` comment blocks within ``/src.ts/``
+- Updating `/docs.wrm/*\*.wrm`
+- Adding links: `/docs.wrm/links/*.txt`
+- Updating API jsdocs: `/*\* ... */` comment blocks within `/src.ts/`
```

Comment:
update `docs.wrm/` to `docs/`

Reply:
Addressed in [f799338](https://github.com/dominant-strategies/quais-6.js/pull/127/commits/f79933813d77192842ed793ec55a5b256e76bac4)